native/jni/magiskboot/bootimg.cpp

```diff
@@ -761,7 +761,11 @@ void repack(const char *src_img, const char *out_img, bool skip_comp) {
         memcpy(footer, boot.avb_footer, sizeof(AvbFooter));
         footer->original_image_size = __builtin_bswap64(off.total);
         footer->vbmeta_offset = __builtin_bswap64(off.vbmeta);
-        vbmeta->flags = __builtin_bswap32(3);
+        if(keep_dm_verity) {
```

Comment:
format your code

Reply:
make it a choice in the app like dm verify is difficult for me.

the choice interface only shows when users don't have root. I am not sure if when those who have root chose 'select and patch a file', how magisk get the parameter to decide keep vbmeta verity open or not.